src/js/lib/pixiUiObject.js

```diff
@@ -982,8 +983,25 @@ class PixiDraw extends EventEmitter {
 		this._fontName = val;
 	}
 
+	// Max uses the fontface attribute for both fontWeight and fontStyle, so we must parse the attribute's value
 	set_font_weight(val) {
-		this._fontWeight = (val === "regular") ? "" : val;
+		console.log(val);
+		if (val == "regular") {
```

Comment:
we should use === instead of == here in order to include type in the equality check